Add release notes for release 0.8.1 (#1096)
standard-for-public-code/standard-for-public-code: RELEASE_NOTES.md

---
# Version history

## Version 0.8.1

> April 11th 2025: 🧑‍🤝‍🧑 The 18th draft changes to community governance model.

Reviewer: I see that we have uncovered another oversight in the releasing documentation!

The date should be `DATE-OF-RELEASE`, so that it can be [set by the automation](https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/script/update-release-notes-date.sh#L46).

Author: Aha! Perhaps this is correct?

```suggestion
DATE-OF-RELEASE: 🧑‍🤝‍🧑 The 18th draft changes to community governance model.
```

Author: Typo fixed in https://github.com/standard-for-public-code/standard-for-public-code/pull/1096/commits/7c12ccc36c7cc2bf498f5ca875874e5626c3cda0

* The project has switched to a community governance model.

>   * Governance model new covers steering team, composition, responsibilities, meetings, decision making process, lazy consensus, voting, and code of Conduct.

Reviewer: s/new/now/

>   * Governance model new covers steering team, composition, responsibilities, meetings, decision making process, lazy consensus, voting, and code of Conduct.

Reviewer 2: ```suggestion
  * Governance model now covers steering team, composition, responsibilities, meetings, decision making process, lazy consensus, voting, and code of Conduct.
```

>   * The website is relocated to a new domain [https://standard.publiccode.net/](https://www.standardforpubliccode.org/).

Reviewer: ```suggestion
  * The website is now located at the new domain [https://standard.publiccode.net/](https://www.standardforpubliccode.org/).
```

Author: Adjusted the website url to the new one.

  * SPDX copyright text updated for all project files.
* Removed optional requirement in Document the code about tempting examples.

> * Critera texts has been slightly simplified, to be more easily understood, with less technical terms.

Reviewer 2: ```suggestion
* Criteria texts has been slightly simplified, to be more easily understood, with less technical terms.
```
Another small typo

Reviewer: also, "Criteria texts _has_ been" should be "Criteria texts _have_ been"
